src/model/channel/message.rs

```diff
@@ -154,30 +152,13 @@ impl Message {
     ///
     /// # Errors
     ///
-    /// If the `cache` is enabled, returns a [`ModelError::InvalidPermissions`] if the current user
-    /// does not have the required permissions.
-    ///
     /// Returns a [`ModelError::MessageAlreadyCrossposted`] if the message has already been
     /// crossposted.
     ///
     /// Returns a [`ModelError::CannotCrosspostMessage`] if the message cannot be crossposted.
     ///
     /// [Manage Messages]: Permissions::MANAGE_MESSAGES
     pub async fn crosspost(&self, cache_http: impl CacheHttp) -> Result<Message> {
```

Comment:
This can be `&Http` too.
```suggestion
    pub async fn crosspost(&self, http: &Http) -> Result<Message> {
```